src/test/java/com/google/sps/DatastorePlaceGuideRepositoryTest.java

```diff
@@ -0,0 +1,207 @@
+package com.google.sps;
+
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.RepositoryType;
+import com.google.sps.placeGuide.PlaceGuide;
+import com.google.sps.placeGuide.repository.PlaceGuideRepository;
+import com.google.sps.placeGuide.repository.PlaceGuideRepositoryFactory;
+import com.google.sps.placeGuide.repository.impl.DatastorePlaceGuideRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.ArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(JUnit4.class)
+public final class DatastorePlaceGuideRepositoryTest{
```

Comment:
Add a few tests in which several entities are stored in the database, and only some of them fulfill the requirements of the query. Testing with a single PlaceGuide is not comprehensive. 
To simplify your work, create ~3 PlaceGuides at the beginning, you could implement the following functions:  
- a function which transforms a PlaceGuide into an entity
- one which saves an entity in the database 
- one which saves the PlaceGuides from a List in the database
- one which compares the content of two PlaceGuide lists

Reply:
done.

Reply:
Looks good! :) thanks!